packages/react-log-viewer/src/LogViewer/LogViewerCustomControl.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import { LogViewerContext } from './LogViewerContext';
+import { Button, ToolbarItem } from '@patternfly/react-core';
+import { ButtonProps } from '@patternfly/react-core/src';
+
+export interface LoggerCustomControlProps extends Omit<ButtonProps, 'onClick'> {
+  /* Icon for the user defined button */
```

Comment:
Should still be updated to use `/** text */` to have the docs pick up the descriptions.